### ISSUE TITLE
fullscreen as a webapp on ios

### ISSUE THREAD

**Issue by crowell** [magne4000/quassel-webserver#259]
ios requires some fancy tags for fullscreen webapps
adding this would fix it https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html

```diff
diff --git a/views/index.pug b/views/index.pug
index a313a2f..72912b4 100644
--- a/views/index.pug
+++ b/views/index.pug
@@ -1,6 +1,7 @@
 doctype html
 html(ng-app="quassel", ng-controller="ConfigController")
     head
+      meta(name='apple-mobile-web-app-capable', content='YES')
       meta(charset='utf-8')
       title Quassel Web App
       meta(http-equiv='X-UA-Compatible', content='IE=edge')
```

**Comment by crowell** [comment]
a few more changes that makes it more "mobile-friendly" on ios at least.
prevents the page from being scrolled around, and adds the "kinetic-scrolling" to the nick and channel divs

```diff
diff --git a/public/stylesheets/mobile.css b/public/stylesheets/mobile.css
new file mode 100644
index 0000000..0cd962e
--- /dev/null
+++ b/public/stylesheets/mobile.css
@@ -0,0 +1,13 @@
+body, html
+{
+       overflow:hidden;
+       width:100%;
+       overflow-x: hidden;
+       overflow-y: hidden;
+       position: fixed;
+}
+
+div
+{
+       -webkit-overflow-scrolling: touch; /* Lets it scroll lazy */
+}
diff --git a/views/index.pug b/views/index.pug
index a313a2f..2fae43a 100644
--- a/views/index.pug
+++ b/views/index.pug
@@ -1,13 +1,15 @@
 doctype html
 html(ng-app="quassel", ng-controller="ConfigController")
     head
+      meta(name='apple-mobile-web-app-capable', content='YES')
       meta(charset='utf-8')
       title Quassel Web App
       meta(http-equiv='X-UA-Compatible', content='IE=edge')
       meta(name='description', content='Quassel Web App')
       meta(name='author', content='Joël Charles')
       meta(name='author', content='Johannes Siipola')
-      meta(name="viewport", content="width=device-width, initial-scale=1")
+      meta(name="viewport", content="width=device-width, initial-scale=1, user-scalable=no, height=device-height")
       link(rel='icon', href='favicon.ico')
       style.
         [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .splash {
@@ -17,6 +19,7 @@ html(ng-app="quassel", ng-controller="ConfigController")
           display: block !important;
         }
       link(rel='stylesheet', href=settings.prefix('/stylesheets/fonts.css'))
+      link(rel='stylesheet', href=settings.prefix('/stylesheets/mobile.css'))
       link(rel='stylesheet', href=settings.prefix('/stylesheets/theme-' + (settings.val.default.theme || settings.val.theme) + '.css'), theme)
     body.login-page
       include ./ng-templates/modalJoinChannel
```

**Comment by magne4000** [comment]
Thanks! It's available in `3.0`
